packages/realm-react/CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contribution Guide
+## Introduction
+We are extremely happy for any contributions to `@realm/react`.  Realm is a very busy team and has many things to work on.  If you are having an issue with this package and want to take it on yourself, feel free to open a PR and we will help guide you to get it merged.
+
+That being said, the following should be considered along with the code changes:
+* Regression tests that fail without the code changes and pass with the code changes
+  * Make sure the tests suite is passing before writing any code or new tests
+  * It is recommended to write a failing tests first, before making any changes
+* Run the `eslint` and `tsc` before opening your PR for review
+  * `npm run build`
+  * `npm run lint`
+* Add comments explaining any non-obvious changes
+* If you are making changes to the interfaces, please update the tsdoc blocks as well
+* Please consult the general [realm-js contribution guide](https://github.com/realm/realm-js/blob/master/CONTRIBUTING.md)
+
+
+## Building and Running the Tests
+`realm-js` is a monorepo that uses `lerna` to dynamically link dependencies.  In the case of `@realm/react`, the `realm` library is linked into the package.  Therefore, one must build `realm` in order to run the `@realm/react` tests.  There is already a [guide for building realm](https://github.com/realm/realm-js/blob/master/contrib/building.md), but an abridged version is provided here.
+
+It is assumed you are already setup to run a `react-native` project and are on a mac (if not see the [guide for building realm](https://github.com/realm/realm-js/blob/master/contrib/building.md)), so a good amount of the required [pre-req steps](https://github.com/realm/realm-js/blob/master/contrib/building.md#setup-instructions-for-macos) are already accomplished.  On requirement is `cmake`, which can be installed with:
```

Comment:
```suggestion
It is assumed you are already setup to run a `react-native` project and are on a mac (if not see the [guide for building realm](https://github.com/realm/realm-js/blob/master/contrib/building.md)), so a good amount of the required [pre-req steps](https://github.com/realm/realm-js/blob/master/contrib/building.md#setup-instructions-for-macos) are already accomplished.  One requirement is `cmake`, which can be installed with:
```

Reply:
Perhaps the guide on building the root package should be simplified instead? It feels a bit wet (`!dry`) to repeat it here. It might very well change in the future and it would be good to simply update it once.

Reply:
That's a good point.  I'll leave it as is for now, but I agree, that we should simplify the building instructions for `realm` and then just link to those instructions from here.